submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>dropdown</title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<nav>
+    <ul class="navBar">
+        <li class="navBar__item item_txt"><a href="#">Mail</a></li>
+        <li class="navBar__item item_txt"><a href="#">Images</a></li>
+        <li class="navBar__item item_btn">
+            <input id="input_1" type="checkbox">
+            <label for="input_1" id="label_1">
+                <img alt="dropdown menu" src="icons/dropdown_menu.png">
+            </label>
+            <nav class="dropdownMenu">
```

Comment:
No need to include a `nav` tag inside another one, all content inside `nav` is navigation)

Reply:
Left just a general nav.